client/client.go

```diff
@@ -262,3 +264,38 @@ func (c *Client) Status(serial string) (hostmgr.Host, error) {
 
 	return host, fmt.Errorf("host %s not found.", serial)
 }
+
+func (c *Client) GetConfig() (string, error) {
+	resp, err := http.Get(fmt.Sprintf("%s://%s:%d/admin/mayu_config", c.Scheme, c.Host, c.Port))
+	if err != nil {
+		return "", err
+	}
+	if resp.StatusCode > 399 {
+		return "", fmt.Errorf("invalid status code '%d'", resp.StatusCode)
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+
+	return string(body), nil
+}
+
+func (c *Client) SetConfig(conf pxemgr.Configuration) (error) {
```

Comment:
`(error)` should be `error`, which probably gets fixed automatically as well when you run `gofmt`.